xclim/testing/tests/test_ensembles.py

```diff
@@ -145,19 +145,19 @@ def test_calc_perc(self, transpose):
             ens = ens.transpose()
 
         out1 = ensembles.ensemble_percentiles(ens, split=True)
-        np.testing.assert_array_equal(
+        np.testing.assert_array_almost_equal(
             mquantiles(
                 ens["tg_mean"].isel(time=0, lon=5, lat=5), 0.1, alphap=1, betap=1
             ),
             out1["tg_mean_p10"].isel(time=0, lon=5, lat=5),
         )
-        np.testing.assert_array_equal(
+        np.testing.assert_array_almost_equal(
```

Comment:
The error comes from an unpacking of scalar values I have added, I will try to fix this.

Reply:
Oh no it's not the unpacking, I don't get why there is ~10e-14 difference between the two...

Reply:
That is very small, could it be just some float32-float64 conversion?

Reply:
Yeah you are right, `ens["tg_mean"].dtype` is float32 but `mquantiles` result is float64 so it is very likely a conversion issue.
Thanks @Zeitsperre for the fix!